src/sentry/static/sentry/app/views/eventsV2/table/queryField.tsx

```diff
@@ -359,6 +359,31 @@ class QueryField extends React.Component<Props> {
     return inputs;
   }
 
+  renderTag(kind) {
+    let text, tagType;
+    switch (kind) {
+      case FieldValueKind.FUNCTION:
+        text = 'f(x)';
```

Comment:
All the other options use words, so maybe this one should too? Plus we have functions like `f()`. Or did you choose `f(x)` because the word `function` is too long?

```suggestion
        text = 'function';
```

Reply:
To both match design mocks, and cause we have `f(x)` in the explanation at the top of the modal:
<img width="706" alt="image" src="https://user-images.githubusercontent.com/4205004/100799581-4b665800-33f3-11eb-980e-2e837933123c.png">